src/core_plugins/metric_vis/public/metric_vis_params.html

```diff
@@ -112,8 +112,8 @@
             <i class="fa fa-danger text-danger"></i>
             <span
               i18n-id="metricVis.params.ranges.warning.specifyRangeDescription"
-              i18n-default-message="{strongRequiredDescripion} You must specify at least one range."
-              i18n-values="{ strongRequiredDescripion: '<strong>' + editorState.requiredDescription + '</strong>' }"
+              i18n-default-message="{strongRequiredDescription} You must specify at least one range."
```

Comment:
Please rename this variable `strongRequiredDescription` it is bad to use tag name for variables.
Just requiredDescription

Reply:
+1, please never use tag name in the id (I guess it's something rather old and we don't do that anymore)